desc/magnetic_fields.py

```diff
@@ -1058,53 +1197,64 @@ def compute_magnetic_field(self, coords, params=None, basis="rpz", grid=None):
         return B
 
     @classmethod
-    def from_mgrid(cls, mgrid_file, extcur=1, method="cubic", extrap=False):
+    def from_mgrid(cls, mgrid_file, extcur=None, method="cubic", extrap=False):
```

Comment:
I changed this so that now by default it will use the coil currents from the mgrid file itself, without the user having to pass them in. This is the typical case for mgrid files in "raw" mode. 

I added a check so that mgrid files in "scaled" mode will throw a `UserWarning` if the currents are not given.